helix-term/src/application.rs

```diff
@@ -32,19 +36,41 @@ pub type LspCallback =
 pub type LspCallbacks = FuturesUnordered<LspCallback>;
 pub type LspCallbackWrapper = Box<dyn FnOnce(&mut Editor, &mut Compositor) + Send>;
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Configuration {
+    pub theme: Option<String>,
```

Comment:
I think this is good and can also hold the keymap from #268 later, \cc @PabloMansanet